packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx

```diff
@@ -103,7 +97,7 @@ it('displays events cards or placeholder if there are no events', () => {
 });
 
 describe('the past events card', () => {
-  const events = createListEventResponse(3).items;
+  const events = createListEventResponse(4).items;
```

Comment:
This seems weird. Why are we giving 4 events and checking three are displayed?

Reply:
The 'View All' link for past events wasn't checking for the number os past events, therefore it was showing it no matter how many results were being returned. I added the `pastEvents && pastEvents.length > 3` check so it could behave like the others and then it has to be tested receiving 3+ results in order to make sure the link is being rendered.

Reply:
I'll add the fourth one for the first test.